spec/requests/monthly_report_comment_spec.rb

```diff
@@ -0,0 +1,38 @@
+describe MonthlyReportCommentsController, type: :request do
+  let(:user) { create(:user) }
+  let(:report) { create(:monthly_report) }
+
+  before { login user }
+
+  describe '#create POST /monthly_report_comments' do
+    let(:comment) { build(:monthly_report_comment, monthly_report: report) }
+    let(:comment_params) { { comment: comment.attributes.slice('comment', 'monthly_report_id') } }
+
+    before { post monthly_report_comments_path, comment_params }
+
+    it { expect(response).to have_http_status :success }
+    it { expect(user.monthly_report_comments.size).to eq 1 }
+    it { expect(user.monthly_report_comments.first.comment).to eq comment.comment }
+  end
+
+  describe '#update PATCH /monthly_report_comments/:id' do
+    let!(:comment) { create(:monthly_report_comment, monthly_report: report, user: user) }
+    let(:after_comment) { build(:monthly_report_comment, monthly_report: report) }
```

Comment:
__[rubocop]__
* Line is too long. [82/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)